tests/test_fyle/test_task.py

```
@@ -369,3 +374,819 @@ def test_skip_expenses_and_post_accounting_export_summary(mocker, db):
+    # Mock the expense data to have different fund source
+    mock_expense_data = {
+        'id': expense.expense_id,
+        'source_account_type': 'PERSONAL_CORPORATE_CREDIT_CARD_ACCOUNT'  # Different from current
+    }
+
+    mock_handle_fund_source_changes = mocker.patch(
+        'apps.fyle.tasks.handle_fund_source_changes_for_expense_ids',
+        return_value=None
+    )
+
+    update_non_exported_expenses([mock_expense_data], workspace_id=1)
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'workspace_id' in function call

(E1123)